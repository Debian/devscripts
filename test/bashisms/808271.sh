#!/bin/sh

if type which; then
  echo "use which"
fi

if ! type which; then
  echo "well, idk"
fi
