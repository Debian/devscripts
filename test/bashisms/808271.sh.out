possible bashism in bashisms/808271.sh line 3 (type):
if type which; then
possible bashism in bashisms/808271.sh line 7 (type):
if ! type which; then
